lib/addons/ebs-csi-driver/iam-policy.ts

```diff
@@ -1,145 +1,147 @@
-export const AmazonEksEbsCsiDriverPolicy = {
-    "Version": "2012-10-17",
-    "Statement": [
-    {
-        "Effect": "Allow",
-        "Action": [
-            "ec2:CreateSnapshot",
-            "ec2:AttachVolume",
-            "ec2:DetachVolume",
-            "ec2:ModifyVolume",
-            "ec2:DescribeAvailabilityZones",
-            "ec2:DescribeInstances",
-            "ec2:DescribeSnapshots",
-            "ec2:DescribeTags",
-            "ec2:DescribeVolumes",
-            "ec2:DescribeVolumesModifications"
-        ],
-        "Resource": "*"
-    },
-    {
-        "Effect": "Allow",
-        "Action": [
-            "ec2:CreateTags"
-        ],
-        "Resource": [
-            "arn:aws:ec2:*:*:volume/*",
-            "arn:aws:ec2:*:*:snapshot/*"
-        ],
-        "Condition": {
-            "StringEquals": {
-                "ec2:CreateAction": [
-                    "CreateVolume",
-                    "CreateSnapshot"
+export const AmazonEksEbsCsiDriverPolicy = (partition: string): object => {
```

Comment:
Since it is a function now let's name it accordingly:

```typescript
export function getEbsDriverPolicyDocument(partition: string) : PolicyDocument 
```

Otherwise it is confusing to see some that is named like a constant used with arguments.